### DISCUSSION TITLE
Welcome to the digiRunner-Open-Source Community! Let’s Collaborate and Grow Together! 🚀

### DISCUSSION THREAD

**Discussion by TPIsoftwareOSPO** [TPIsoftwareOSPO/digiRunner-Open-Source#11] [Announcements]
**Hello Everyone! 🎉**

We’re absolutely thrilled to welcome you to the digiRunner-Open-Source community! This is an inclusive and collaborative space for developers, users, and anyone passionate about open-source technology. Whether you’re new to the project or an experienced contributor, we’re excited to have you here, your presence enriches our mission.

**💬 How You Can Get Involved**
Here’s how you can participate and make the most of our community:
**1.  Have Questions or Need Help?**
Post your queries in the Q&A category and get support from the community or our team.
**2.  Share Your Ideas and Suggestions**
Got creative feature requests or feedback? Share them in the Ideas category. Your input matters!
**3.  Showcase Your Work**
If you’re already using digiRunner, we’d love to hear about your experiences and success stories. Share them in the Show and Tell section!

**🌈 Our Community Values**
**1.  Respect and Inclusivity**: Everyone is welcome here, regardless of background, expertise, or experience. Be kind and supportive to each other.
**2.  Open Collaboration**: Share your thoughts, learn from others, and work together to improve the project.
**3.  Innovation and Learning**: This is a place for new ideas, growth, and technical learning—feel free to explore and innovate!

**🚀 Take Your First Step**
**1.  Introduce Yourself**: Leave a comment below to say hello and let us know a little about you!
**2.  Start a Discussion**: Don’t hesitate to ask questions, propose ideas, or share your experiences—your voice matters here.

If you’re unsure where to start, just drop a comment below. Our team and community members are here to help you get onboarded.

Thank you for being part of this journey! Together, let’s shape the future of digiRunner-Open-Source and build something amazing. 🌟

Best regards,
Team digiRunner-Open-Source
2024/12/16

**Comment by BenSFY** [comment]
Honored to be the first one here :) Good to have the open sorced enterprise-grade APIM project

**Reply by TPIsoftwareOSPO** [reply]
Thank you for participating in the digiRunner-Open-Source community discussion! 🎉 Your participation is very important to us, and we look forward to communicating and learning with you to make this project better. If you have any questions or suggestions, please feel free to ask, we are happy to help! 😊

**Comment by Lifetime525** [comment]
Great!! I can use this to build my project... will dive into it. Thanks!!

**Reply by TPIsoftwareOSPO** [reply]
Thank you for your kind words! 😊 We're thrilled to hear that digiRunner will be helpful for your project. If you have any questions or run into anything while diving in, feel free to ask here or open an issue. We’d love to hear more about your project and how digiRunner fits in—keep us posted! 🚀
